examples/text_to_mscoco.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+from pathlib import Path
+import os
+
+from munch import munchify
+from torchvision.utils import save_image
+
+from latent_diffusion import get_solver
+from latent_sdxl import get_solver as get_solver_sdxl
+from utils.callback_util import ComposeCallback
+from utils.log_util import create_workdir, set_seed
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Latent Diffusion")
+    parser.add_argument("--workdir", type=Path, default="examples/workdir/mscoco")
+    parser.add_argument('--prompt_dir', type=Path, default=Path('/home/user/research/dataset/mscoco/coco_v2.txt'))
```

Comment:
Would you remove the default path?

Reply:
or add relative path

Reply:
Agreed. Corrected in [6270aef](https://github.com/CFGpp-diffusion/CFGpp/pull/8/commits/6270aef3c01d4b3e47c83d80a4e407a235f72ae1)